grpc-proto/src/main/java/io/stargate/grpc/StargateBearerToken.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.stargate.it.grpc;
+package io.stargate.grpc;
```

Comment:
Isn't this only used in integration tests?

Reply:
It can also be used by application integrating with `grpc-proto`